src/esp/scene/SemanticScene.h

```diff
@@ -130,9 +133,9 @@ class SemanticScene {
    */
   static bool loadHM3DHouse(const std::string& filename,
                             SemanticScene& scene,
-                            CORRADE_UNUSED const quatf& rotation =
-                                quatf::FromTwoVectors(-vec3f::UnitZ(),
-                                                      geo::ESP_GRAVITY));
+                            CORRADE_UNUSED const Magnum::Quaternion& rotation =
```

Comment:
A drive-by fix again -- what's the unused macro here for? It should be only needed in the definition.